src/main/java/app/Methods.java

```diff
@@ -7,4 +7,5 @@
     public static MethodApplier GET_WITH_BODY = (Request request) -> "some body";
     public static MethodApplier SIMPLE_OPTIONS = (Request request) -> "";
     public static MethodApplier ECHO_BODY = (Request request) -> request.body();
+    public static MethodApplier REDIRECT = (Request request) -> "redirect: /simple_get";
```

Comment:
I'm kinda late to the party when it comes to reviewing this, but with this new design, how would you go about adding response headers?

Reply:
Right now the header fields associated with response statuses are defined in `ResponseBuilder`, but there isn't a way for the user to add custom headers in these controller-action style methods.